speech/cloud-client/beta_snippets.py

```diff
@@ -279,28 +283,20 @@ def transcribe_file_with_word_level_confidence(speech_file):
         description=__doc__,
         formatter_class=argparse.RawDescriptionHelpFormatter)
     parser.add_argument('command')
-    parser.add_argument(
-        'path', help='File for audio file to be recognized')
-    parser.add_argument(
-        'first', help='First language in audio file to be recognized',
-        nargs='?')
-    parser.add_argument(
-        'second', help='Second language in audio file to be recognized',
-        nargs='?')
 
     args = parser.parse_args()
 
     if args.command == 'enhanced-model':
-        transcribe_file_with_enhanced_model(args.path)
+        transcribe_file_with_enhanced_model()
     elif args.command == 'metadata':
-        transcribe_file_with_metadata(args.path)
+        transcribe_file_with_metadata()
     elif args.command == 'punctuation':
-        transcribe_file_with_auto_punctuation(args.path)
+        transcribe_file_with_auto_punctuation()
     elif args.command == 'diarization':
-        transcribe_file_with_diarization(args.path)
+        transcribe_file_with_diarization()
     elif args.command == 'multi-channel':
-        transcribe_file_with_multichannel(args.path)
+        transcribe_file_with_multichannel()
     elif args.command == 'multi-language':
-        transcribe_file_with_multilanguage(args.path, args.first, args.second)
```

Comment:
For future reference, argparse's [sub-commands feature](https://docs.python.org/2/library/argparse.html#sub-commands) would be helpful to avoid having args that only matter for one command or another.

Reply:
Sounds good.

Reply:
Sounds good.